src/hooks/useConnectRelatedVenues.ts

```diff
@@ -170,37 +171,37 @@ export const useConnectRelatedVenues: ReactHook<
   const makeEventsQueryConfig = (
     doc: string,
     storeAs: string
-  ): SparkleRFQConfig =>
+  ): SparkleDocRFQSetting =>
```

Comment:
What's the intent of `SparkleDocRFQSetting` here compared to `SparkleRFQSetting`?

Reply:
It's to ensure that if your intention is to query for a single doc, you only query for that doc.

Reply:
From @thecadams on zoom (paraphrased): essentially we had times when a `doc` that we were supplying was `undefined`, and so we were loading the entire collection (because it's an optional param in the underlying `useFirestoreConnect` hook), when our intent was to load a single tem or bailout.